ArduPlane/commands_logic.cpp

```diff
@@ -891,7 +891,7 @@ void Plane::do_change_speed(const AP_Mission::Mission_Command& cmd)
     switch (cmd.content.speed.speed_type)
     {
     case 0:             // Airspeed
-        if (cmd.content.speed.target_ms > 0) {
+        if ((cmd.content.speed.target_ms >= aparm.airspeed_min.get()) && (cmd.content.speed.target_ms <= aparm.airspeed_max.get()))  {
```

Comment:
need to make do_change_speed a bool fn, and fail the command_long if rejected

Reply:
fixed